src/main/java/teammates/ui/controller/AdminEmailComposeSendAction.java

```diff
@@ -145,7 +135,7 @@ private void moveJobToAddressModeTaskQueue() {
         }
         taskQueuer.scheduleAdminEmailPreparationInAddressMode(emailId, addressReceiverListString);
 
-        statusToAdmin = "Email queued for sending." + "<br/>" + "Receipient: " + addressReceiverListString;
+        statusToAdmin += "<br/>" + "Receipient: " + addressReceiverListString;
```

Comment:
The `statusToAdmin` is initialised to `Email queued for sending.` in this case (by default, it is not). Good catch to the typo. :P